documentation/src/main/asciidoc/titles/developing/stories.adoc

```diff
@@ -2,5 +2,8 @@
 //Please review authoring guidelines in the Contributors Guide.
 
 include::{stories}/assembly_developing.adoc[leveloffset=+1]
-include::{stories}/assembly_extending.adoc[leveloffset=+1]
 include::{stories}/assembly_custom_interceptors.adoc[leveloffset=+1]
```

Comment:
this can be removed IMO. 
They are deprecated in 10: `@deprecated Since 10.0, custom interceptors support will be removed and only modules will be able to define interceptors`
If you don't want to remove them, just put a warning saying it will be removed... soon... :)

Reply:
thanks, I've added an [IMPORTANT] note to make sure it's clear custom interceptors are deprecated.

Reply:
I think we need to keep if it's deprecated but not yet removed.